DemoApps/SearchWithMaps/app/build.gradle

```diff
@@ -41,7 +41,7 @@ android {
 
 dependencies {
     implementation "com.mapbox.search:mapbox-search-android-ui:1.0.0-rc.5"
-    implementation "com.mapbox.maps:android:10.14.0-beta.1"
+    implementation "com.mapbox.maps:android:11.0.0"
```

Comment:
This change is not needed. You can bump it only together with `com.mapbox.search:mapbox-search-android-ui` dependency.

Note that `DemoApps` is a folder with external demo apps, its sources are not part of the SDK

Reply:
Reverted